appveyor.yml

```diff
@@ -1,15 +1,16 @@
 environment:
-  nodejs_version: "10"
```

Comment:
This is 12 now after merging https://github.com/apiaryio/dredd/pull/1495

Reply:
Will be updated after this branch is rebased against the latest `master`.